classes/02class/exercises/c02-network07/andreazzabr/c02-network07.txt

```diff
@@ -0,0 +1,24 @@
+c02-network07.txt 
+
+Then post your answers for the following:
+
+How did you accomplish both instructions?
+Aws console - vpc - route tables - new route table - Name tag - vpc.
+Select RT - Routes tab - edit routes - add route - 0.0.0.0/0 - target - select gateway - save.
+
+
+Why did you add 0.0.0.0 route to the IGW on public route table?
+So my public hosts can access internet
+
+Why did you add 0.0.0.0 route to the NGW on private route table?
+So my private hosts can access internet
+
+What is the difference of IGW and NGW?
+IGW gives access to public-hosts to the internet
+NGW gives access to private-hosts to the internet
+
+Can you delete the destination route to your VPC network? Why?
+Local network? No, otherwise you can’t reach any other machine locally.
```

Comment:
It means the default route added that maps to your local network. 

You cannot delete it in AWS.

From https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Route_Tables.html

Every route table contains a local route for communication within the VPC. This route is added by default to all route tables. If your VPC has more than one IPv4 CIDR block, your route tables contain a local route for each IPv4 CIDR block. If you've associated an IPv6 CIDR block with your VPC, your route tables contain a local route for the IPv6 CIDR block. You cannot modify or delete these routes in a subnet route table or in the main route table.